articles/article-055-turing-completeness.md

```diff
@@ -0,0 +1,27 @@
+<!-- BEGIN TITLE -->
+# Is x a more powerful programming language than y?
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+There are hundreds or even thousands of programming languages, but are all of these equally powerful, i.e. can they be used to compute an arbitrary problem? The answer is yes--if the language is Turing complete.
```

Comment:
Not sure about the format of the double dash "--". You can say:
Yes, if they are Turing complete.